### PR TITLE
install_NEO_OCL_driver: Fixed ShellCheck errors

### PR DESCRIPTION
**Pull request by artanokhov**
### Details:
 - Fixed ShellCheck errors.

### CODE REVIEW

File: scripts/install_dependencies/install_NEO_OCL_driver.sh [hunk 1]
```diff
@@ -15,7 +15,6 @@
 #
```
Comment (ilya-lavrenov) [inline scripts/install_dependencies/install_NEO_OCL_driver.sh:15]: please, remove `scripts/install_dependencies/What`

Comment (artanokhov) [reply scripts/install_dependencies/install_NEO_OCL_driver.sh:15]: Already in this PR

File: scripts/install_dependencies/install_NEO_OCL_driver.sh [hunk 2]
```diff
@@ -383,8 +378,8 @@ _check_distro_version()
 {
     if [[ $DISTRO == redhat ]]; then
         RHEL_MINOR_VERSION_SUPPORTED="[3-7]"
-        RHEL_VERSION=$(grep -m1 'VERSION_ID' /etc/os-release | grep -Eo "8.${RHEL_MINOR_VERSION_SUPPORTED}")
-        if [[ $? -ne 0 ]]; then
+        cmd=$(grep -m1 'VERSION_ID' /etc/os-release | grep -Eo "8.${RHEL_MINOR_VERSION_SUPPORTED}")
+        if ! cmd; then
```
Comment (dtrawins) [inline scripts/install_dependencies/install_NEO_OCL_driver.sh:382]: That command is not working. Should it be replaced with this:
```suggestion
        if [ -z "$cmd" ]; then
```

File: scripts/install_dependencies/install_NEO_OCL_driver.sh [hunk 3]
```diff
@@ -203,12 +199,12 @@ _uninstall_user_mode_redhat()
...
         echo "rpm -qa | grep $package"
         found_package=$(rpm -qa | grep "$package")
-        if [[ $? -eq 0 ]]; then
+        if [[ ${#found_package} -gt 0 ]]; then
             echo "Found installed user-mode driver, performing uninstall..."
             cmd="rpm -e --nodeps ${found_package}"
             echo "$cmd"
             eval "$cmd"
-            if [[ $? -ne 0 ]]; then
+            if ! eval "$cmd"; then
```
Comment (Wovchena) [inline scripts/install_dependencies/install_NEO_OCL_driver.sh:207]: Remove `eval "$cmd"` one line above

**Review by evgenytalanin-intel — APPROVED** [review]

**Pull request merged**